tests/io/lobster/test_outputs.py

```diff
@@ -649,8 +649,8 @@ def test_attributes(self):
         atomlist = ["O1", "Mn2"]
         types = ["O", "Mn"]
         num_atoms = 2
-        assert charge_Mulliken == self.charge2.Mulliken
-        assert charge_Loewdin == self.charge2.Loewdin
+        assert charge_Mulliken == self.charge2.mulliken
```

Comment:
@JaGeo @QuantumChemist Can you please help me have a look at this LOBTSER change? A lot of test code access deprecated attribute, I wish it's not intended (to test deprecated API)

```
tests/io/lobster/test_lobsterenv.py: 206 warnings
  /home/runner/work/pymatgen/pymatgen/src/pymatgen/io/lobster/lobsterenv.py:193: DeprecationWarning: Mulliken is deprecated
  Use `mulliken` instead.
    self.valences = chg.Mulliken

tests/io/lobster/test_lobsterenv.py: 17 warnings
  /home/runner/work/pymatgen/pymatgen/src/pymatgen/io/lobster/lobsterenv.py:195: DeprecationWarning: Loewdin is deprecated
  Use `loewdin` instead.
    self.valences = chg.Loewdin

tests/io/lobster/test_lobsterenv.py::TestLobsterNeighbors::test_get_anion_types
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_lobsterenv.py:341: FutureWarning: get_anion_types is deprecated; use anion_types in pymatgen.io.lobster.lobsterenv instead.
    assert self.chem_env_lobster0_second.get_anion_types() == {Element("O")}

tests/io/lobster/test_outputs.py::TestCharge::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:652: DeprecationWarning: Mulliken is deprecated
  Use `mulliken` instead.
    assert charge_Mulliken == self.charge2.Mulliken

tests/io/lobster/test_outputs.py::TestCharge::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:653: DeprecationWarning: Loewdin is deprecated
  Use `loewdin` instead.
    assert charge_Loewdin == self.charge2.Loewdin

tests/io/lobster/test_outputs.py: 4128 warnings
  /home/runner/work/pymatgen/pymatgen/src/pymatgen/io/lobster/outputs.py:1680: PendingDeprecationWarning: the matrix subclass is not the recommended way to represent matrices or deal with linear algebra (see https://docs.scipy.org/doc/numpy/user/numpy-for-matlab-users.html). Please adjust your code to use regular ndarray.
    self.band_overlaps_dict[spin]["matrices"].append(np.matrix(overlaps))

tests/io/lobster/test_outputs.py::TestBandoverlaps::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:1494: DeprecationWarning: bandoverlapsdict is deprecated
  Use `band_overlaps_dict` instead.
    bo_dict = self.band_overlaps1.bandoverlapsdict

tests/io/lobster/test_outputs.py::TestBandoverlaps::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:1496: DeprecationWarning: bandoverlapsdict is deprecated
  Use `band_overlaps_dict` instead.
    assert self.band_overlaps1_new.bandoverlapsdict[Spin.up]["max_deviations"][10] == approx(0.0640933)

tests/io/lobster/test_outputs.py::TestBandoverlaps::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:1498: DeprecationWarning: bandoverlapsdict is deprecated
  Use `band_overlaps_dict` instead.
    assert self.band_overlaps1_new.bandoverlapsdict[Spin.up]["matrices"][10].item(-1, -1) == approx(1.0)

tests/io/lobster/test_outputs.py::TestBandoverlaps::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:1500: DeprecationWarning: bandoverlapsdict is deprecated
  Use `band_overlaps_dict` instead.
    assert self.band_overlaps1_new.bandoverlapsdict[Spin.up]["matrices"][10].item(0, 0) == approx(0.995849)

tests/io/lobster/test_outputs.py::TestBandoverlaps::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:1503: DeprecationWarning: bandoverlapsdict is deprecated
  Use `band_overlaps_dict` instead.
    assert self.band_overlaps1_new.bandoverlapsdict[Spin.down]["max_deviations"][9] == approx(0.064369)

tests/io/lobster/test_outputs.py::TestBandoverlaps::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:1505: DeprecationWarning: bandoverlapsdict is deprecated
  Use `band_overlaps_dict` instead.
    assert self.band_overlaps1_new.bandoverlapsdict[Spin.down]["matrices"][9].item(0, -1) == approx(1.37447e-09)

tests/io/lobster/test_outputs.py::TestSitePotentials::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:2211: DeprecationWarning: sitepotentials_Loewdin is deprecated
  Use `sitepotentials_loewdin` instead.
    ]

tests/io/lobster/test_outputs.py::TestSitePotentials::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:2218: DeprecationWarning: sitepotentials_Mulliken is deprecated
  Use `sitepotentials_mulliken` instead.
    ]

tests/io/lobster/test_outputs.py::TestSitePotentials::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:2219: DeprecationWarning: madelungenergies_Loewdin is deprecated
  Use `madelungenergies_loewdin` instead.
    assert self.sitepotential.madelungenergies_Loewdin == approx(-28.64)

tests/io/lobster/test_outputs.py::TestSitePotentials::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:2220: DeprecationWarning: madelungenergies_Mulliken is deprecated
  Use `madelungenergies_mulliken` instead.
    assert self.sitepotential.madelungenergies_Mulliken == approx(-40.02)

tests/io/lobster/test_outputs.py::TestMadelungEnergies::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:2256: DeprecationWarning: madelungenergies_Loewdin is deprecated
  Use `madelungenergies_loewdin` instead.
    assert self.madelungenergies.madelungenergies_Loewdin == approx(-28.64)

tests/io/lobster/test_outputs.py::TestMadelungEnergies::test_attributes
  /home/runner/work/pymatgen/pymatgen/tests/io/lobster/test_outputs.py:2257: DeprecationWarning: madelungenergies_Mulliken is deprecated
  Use `madelungenergies_mulliken` instead.
    assert self.madelungenergies.madelungenergies_Mulliken == approx(-40.02)
```